learning/tour-of-beam/backend/internal/storage/schema.go

```diff
@@ -0,0 +1,99 @@
+// Licensed to the Apache Software Foundation (ASF) under one or more
+// contributor license agreements.  See the NOTICE file distributed with
+// this work for additional information regarding copyright ownership.
+// The ASF licenses this file to You under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance with
+// the License.  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package storage
+
+import (
+	tob "beam.apache.org/learning/tour-of-beam/backend/internal"
+	"cloud.google.com/go/datastore"
+)
+
+// ToB Datastore schema
+// - Content tree has a root entity in tb_learning_path,
+//   descendant entities in tb_learning_module/group/unit have it as a common ancestor
+// - learning path consists of modules, modules consist of either groups or units
+// - A group can have only 1 type of children: only units or only groups
+// - Ordering is established by "order" property
+// - To limit ancestor queries by only first-level descendants, "level" property is used
+
+const (
+	PgNamespace = "Playground"
```

Comment:
Oh.. I remember that :) First i did the same things, namely, wrote constants in the specific package. However, i had package cycling in the future. I moved these constants to the separate package as constants and use them anywhere where it is necessary

Reply:
It has now only that meaning as a Datastore namespace everything lives in
It doesn't have any references outside the Datastore module
If that changes, we make it more general, but not before